packages/iTwinUI-react/src/core/Table/Table.test.tsx

```diff
@@ -389,6 +389,88 @@ it('should handle row clicks', async () => {
   expect(rows[2].classList).toContain('iui-selected');
   expect(onSelect).toHaveBeenCalledTimes(3);
   expect(onRowClick).toHaveBeenCalledTimes(3);
+
+  await user.keyboard('[/ControlLeft][ShiftLeft>]'); // Release Control and Press Shift (without releasing it)
+  await user.click(getByText(mockedData()[0].name)); // Perform a click with `shiftKey: true`
+
+  expect(rows[0].classList).toContain('iui-selected');
+  expect(rows[1].classList).toContain('iui-selected');
```

Comment:
Maybe don't test if all rows are selected. Eg. test if first two rows are selected and last one is not.

Reply:
**[DONE]** It now tests that the 0th and 1st rows is selected but the 2nd one is not